src/comparable.cr

```diff
@@ -1,22 +1,40 @@
 # The `Comparable` mixin is used by classes whose objects may be ordered.
 #
 # Including types must provide an `<=>` method, which compares the receiver against
-# another object, returning a negative number, `0`, or a positive number depending
-# on whether the receiver is less than, equal to, or greater than the other object.
+# another object, returning:
+# - a value less than zero if `self` is less than the other object
+# - a value greater than zero if `self` is greater than the other object
+# - zero if `self` is equal to the other object
```

Comment:
I'd prefer to use the literal `0` instead of "zero", just like `nil`.

Reply:
Done!